### ISSUE TITLE
get focusableElements gets the disabled items as well

### ISSUE THREAD

**Issue by onin-baylen** [zestia/ember-modal-dialog#6]
I have a dialog that has the last element disabled. This only gets enabled when I put something in the textbox within the dialog. Problem is that the focus trapping fails when this button is disabled. This was because it was included in the focusable elements even if it is disabled.

Also found out that :not(disabled) does not work, but :not([disabled]) does.

So Focus trap was ok when I changed the focusable elements into this

  get focusableElements() {
    return [
      ...this.element.querySelectorAll(`
        a[href],
        button:not([disabled]),
        textarea:not([disabled]),
        input:not([disabled]),
        select:not([disabled]),
        [tabindex="0"],
        [contenteditable="true"]
      `)
    ];
  }

**Comment by amk221** [comment]
Thanks, great spot!

**Comment by amk221** [comment]
Fixed in v3
